pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -11031,7 +11031,43 @@ Invalid peptide sequence at position 18
   <data name="EditStaticModDlg_SetModification___0___is_not_a_recognized_Unimod_name_" xml:space="preserve">
     <value>'{0}' is not a recognized Unimod name.</value>
   </data>
+  <data name="MetadataExtractor_ApplyStep_Error_converting___0___to___1___" xml:space="preserve">
+    <value>Error converting '{0}' to '{1}':</value>
+  </data>
+  <data name="MetadataExtractor_ResolveColumn_Unable_to_find_column__0_" xml:space="preserve">
+    <value>Unable to find column {0}</value>
+  </data>
+  <data name="RuleError_ToString_An_error_occurred_applying_the_rule___0___" xml:space="preserve">
+    <value>An error occurred applying the rule '{0}':</value>
+  </data>
   <data name="PublishDocumentDlg_PublishDocumentDlgLoad_Go_to_Tools___Options___Panorama_tab_to_update_the_username_and_password_" xml:space="preserve">
     <value>Go to Tools &gt; Options &gt; Panorama tab to update the username and password.</value>
   </data>
+  <data name="DocumentSettingsDlg_btnDeleteRules_Click_Are_you_sure_you_want_to_delete_the_rule___0___" xml:space="preserve">
+    <value>Are you sure you want to delete the rule '{0}'?</value>
```

Comment:
When you refer to "rule" below, I think you mean "rule set", right? A "rule" is now what you were calling a "rule step"